src/Container/Container.test.tsx

```diff
@@ -12,32 +12,38 @@ describe('<Container />', () => {
 
   it('adds the .container-fluid class', () => {
     const { container } = render(<Container>Content</Container>);
-    const containerElement = container.firstChild;
+    const containerElement = container.firstChild as HTMLElement;
     expect(getClassNames(containerElement)).toContain('container-fluid');
   });
 
   it('adds the .container class', () => {
     const { container } = render(<Container fluid={false}>Content</Container>);
-    const containerElement = container.firstChild;
-    expect(getClassNames(containerElement)).toContain('container');
+    const containerElement = container.firstChild as HTMLElement;
+    expect(getClassNames(containerElement!)).toContain('container');
     expect(getClassNames(containerElement)).not.toContain('container-fluid');
   });
 
-  ['xs', 'sm', 'md', 'lg', 'xl'].forEach((size) => {
+  ['xs', 'sm', 'md', 'lg', 'xl'].forEach((size: ContainerSize) => {
     it(`adds the .container-mw-${size} class`, () => {
       const { container } = render(<Container size={size}>Content</Container>);
-      const containerElement = container.firstChild;
+      const containerElement = container.firstChild as HTMLElement;
       expect(getClassNames(containerElement)).toContain(`container-mw-${size}`);
     });
   });
 
+  it('does not add a size class when size is not specified', () => {
+    const { container } = render(<Container>Content</Container>);
```

Comment:
I'd rather use an explicit `size="full"` than assume that omitting the size means "full width".. Omitting a size should make the container fit its contents, just like a `<div>` without a `max-width`.
I think the bootstrap class `mw-100` does this?

Reply:
I agree with you and tried this approach first.

The problem is that the current component allows the `size` props to be omitted, and it behaves like "full width". You can check it here, removing the `size` from one example: https://paragon-openedx.netlify.app/components/container/

We have many containers in our code base without `size` that probably rely on this, so I don't think we can change that.
So I pushed back the `size={full}` option.

Reply:
I tried to update the API docs here: bedcb9b914d5eb47518f15d9e9d34841bfb37e45

Reply:
> The problem is that the current component allows the size props to be omitted, and it behaves like "full width". You can check it here, removing the size from one example: https://paragon-openedx.netlify.app/components/container/

Passing in `size=""` gives us the fullscreen behaviour you're adding here.. so do we need this change?